x/programs/rust/wasmlanche-sdk/src/program.rs

```diff
@@ -1,5 +1,8 @@
+use std::hash::Hash;
+
 use borsh::{BorshDeserialize, BorshSerialize};
 
+use crate::state::Key;
```

Comment:
nit: 
Can we combine this with the imports below, please?

Reply:
Yes, thanks!